pkg/features/oidc.go

```diff
@@ -31,7 +31,9 @@ func activateKubeadmOIDC(feature *kubeoneapi.OpenIDConnect, args *kubeadmargs.Ar
 	optionalMapSet(args.APIServer.ExtraArgs, "oidc-username-claim", feature.Config.UsernameClaim)
 	optionalMapSet(args.APIServer.ExtraArgs, "oidc-username-prefix", feature.Config.UsernamePrefix)
 	optionalMapSet(args.APIServer.ExtraArgs, "oidc-groups-claim", feature.Config.GroupsClaim)
-	optionalMapSet(args.APIServer.ExtraArgs, "oidc-groups-prefix", feature.Config.GroupsPrefix)
+	if feature.Config.GroupsPrefix != "-" {
```

Comment:
Shouldn't we also do this for `oidc-username-prefix`?

Reply:
No, it's handled by the kubeapi-server

Reply:
The docs at https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/ says:

`--oidc-username-prefix string`
> If  provided, all usernames will be prefixed with this value. If not  provided, username claims other than 'email' are prefixed by the issuer  URL to avoid clashes. To skip any prefixing, provide the value '-'.

Reply:
Can you please add a comment clarifying this?

Reply:
done